_docs/reporting/dora-metrics.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "DORA metrics"
+description: "Get insights for your deployments"
+group: reporting
+toc: true
+---
+
+DevOps is a collaboration paradigm and sometimes it is mistaken for being too
+abstract or too generic. In an effort to quantify the benefits of adopting DevOps,
+[Dora Research](https://www.devops-research.com/research.html#capabilities) (acquired by Google in 2018) has introduced 4 key metrics that
+define specific goals for improving the software lifecycle in companies that are
+interested in adopting DevOps.
+
+These metrics [are defined as](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance):
+
+* Deployment Frequency — How often an organization successfully releases to production
+* Lead Time for Changes — The amount of time it takes a commit to get into production
+* Change Failure Rate — The percentage of deployments causing a failure in production
+* Time to Restore Service — How long it takes an organization to recover from a failure in production
+
+Monitoring these metrics can help you identify delivery issues in your organization as they can detect bottlenecks among teams and show you where you need to optimize your workflows (either on a technical or organizational level).
+
+Codefresh offers support for DORA metrics out of the box. You can visit your metric dashboard by selecting *DORA metrics* from the left sidebar
+
+{% include
+image.html
+lightbox="true"
+file="/images/reporting/dora-metrics.png"
+url="/images/reporting/dora-metrics.png"
+alt="DORA metrics report"
+caption="DORA metrics report"
+max-width="100%"
+%}
+
+## Filters
+
+At the top of the window you have options for filtering all metrics with multi-select controls. These include:
+
+
+* Runtimes: limit metrics to Applications only from selected runtimes 
+* Clusters: limit metrics to Applications deployed to selected cluster
+* Applications: show selected applications 
+* Labels: show only Applications that have the matching labels (key/value) 
+* Time: limit metrics to a specific time period
+
+All filtering components support auto-complete and are multi-select. This means that you can define exactly the subset of applications that you are interested in.
```

Comment:
1. I would combine this with the introductory sentence, like this may be:
Use filters to define the exact subset of applications you are interested in:

2. Not sure we need to mention multi-select and auto-complete.
3. Lines 40 to 44: Can you capitalize the letter after the colon? ( I can do the editing if you prefer)